blocks/api/test/serializer.js

```diff
@@ -1,11 +1,6 @@
 /**
  * External dependencies
  */
-import { expect } from 'chai';
-
-/**
- * WordPress dependencies
- */
 import { createElement, Component } from 'element';
```

Comment:
This section should now be titled `WordPress dependencies`, not `External dependencies`.